test/ship_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship.rb'
+
+class PlayerTest < Minitest::Test
+
+  def test_ship_exists
+    ship = Ship.new("Cruiser", 3)
+
+    assert_instance_of Ship, ship
+  end
+
+  def test_ship_instantiation
+    ship = Ship.new("Cruiser", 3)
+
+    assert_equal "Cruiser", ship.name
+    assert_equal 3, ship.length
+    assert_equal 3, ship.health
+  end
+
+  def test_ship_full_health_has_sunk
+    ship = Ship.new("Cruiser", 3)
+
+    assert_equal false, ship.sunk?
+  end
+
+  def test_ship_can_hit
+    ship = Ship.new("Cruiser", 3)
+
+    ship.hit
+
+    assert_equal 2, ship.health
+  end
+
+  def test_ship_can_hit_and_sink
+    ship = Ship.new("Cruiser", 3)
+
+    ship.hit
```

Comment:
if we're testing the sunk? method, probably should call it after these hits to make sure it isn't returning sunk before it should.

so "assert_equal false, ship.sunk?" here and after the next ship.hit?